core/rio/languages/src/main/java/org/eclipse/rdf4j/rio/languages/BCP47LanguageHandler.java

```diff
@@ -0,0 +1,101 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.
```

Comment:
I can't find any other version of the copyright header in the files I am skimming through. Do you have an example of the correct header to use? (Will require a bulk replacement as this one is used extensively)